frontend/src/metabase/writeback/containers/DataAppContext/DataAppContext.ts

```diff
@@ -1,49 +1,24 @@
 import { createContext, useContext } from "react";
-import _ from "lodash";
-
-import { Column } from "metabase-types/types/Dataset";
-
-type FieldName = string;
-type CardName = string;
-
-type ObjectDetailField = {
-  column: Column;
-  value: unknown;
-};
-
-export type FormattedObjectDetail = Record<FieldName, ObjectDetailField>;
-
-export type FormattedListInfo = { rowCount: { value?: number } };
```

Comment:
these need to be removed from the import in `utils.js`

Reply:
Yup, that's in progress, thanks for pointing out

Reply:
oops. didn't see this was still draft. 😊